code/game/machinery/atmoalter/portable_atmospherics.dm

```diff
@@ -27,7 +27,7 @@
 			update_icon()
 
 /obj/machinery/portable_atmospherics/process()
-	if(!connected_port) //only react when pipe_network will ont it do it for you
+	if(!connected_port) //only react when pipe_network will not it do it for you

```

Comment:
I recall a line from you in the IRC shittalking people who tiptoe around awful stuff in the code and leave it unfixed